src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -263,4 +266,13 @@ private void validateFilingDate(LocalDate filingDate, Errors errors, String logg
             ApiLogger.infoContext(loggingContext, errorMessage);
         }
     }
+
+    private void validateHasSecureRegister(Boolean hasSecureRegister, Errors errors, String loggingContext) {
+        if (Boolean.TRUE.equals(hasSecureRegister) || hasSecureRegister == null) {
+            String qualifiedFieldName = OverseasEntitySubmissionDto.HAS_SECURE_REGISTER_FIELD + ": " + hasSecureRegister;
```

Comment:
Qualified field name should just be the field name (with full path). See implementation for 'has sold land'.

Reply:
I have added the value so that we know what the error really is (true or null)

Reply:
We should know that from the error message. Not always the case, but should be in this situation, as two different messages are required.